packages/http/src/validator/validators/utils.ts

```diff
@@ -3,19 +3,37 @@ import { DiagnosticSeverity } from '@stoplight/types';
 import * as O from 'fp-ts/Option';
 import { pipe } from 'fp-ts/function';
 import { NonEmptyArray, fromArray, map } from 'fp-ts/NonEmptyArray';
-import Ajv, { ErrorObject, Options } from 'ajv';
+import Ajv, { ErrorObject, Logger, Options } from 'ajv';
 import type AjvCore from 'ajv/dist/core';
 import Ajv2019 from 'ajv/dist/2019';
 import Ajv2020 from 'ajv/dist/2020';
 import addFormats from 'ajv-formats';
 import type { JSONSchema } from '../../';
 import { compareDateTime, date_time, fmtDef } from './dateTime';
 
+const unknownFormatSilencer: Logger = {
```

Comment:
can we just call it a logger or unknownFormatSilencerLogger.

Reply:
@radzserg Fixed